src/components/app.tsx

```diff
@@ -39,7 +39,10 @@ export default function App() {
 			{ isWindows() && <WindowsTitlebar className="h-titlebar-win flex-shrink-0" /> }
 			<HStack spacing="0" alignment="left" className="flex-grow">
 				<MainSidebar className="basis-52 flex-shrink-0 h-full" />
-				<main data-testid="site-content" className="bg-white h-full flex-grow rounded-chrome">
+				<main
+					data-testid="site-content"
+					className="bg-white h-full flex-grow rounded-chrome overflow-hidden"
```

Comment:
`overflow-hidden` here ensures that the corners remain rounded. I tested all of the popups we currently have to ensure that this change doesn't result in elements being cut-off, and all seems okay.